17/bullseye/Dockerfile

```
@@ -74,6 +80,12 @@ VOLUME /home/${user}/.jenkins
 VOLUME ${AGENT_WORKDIR}
 WORKDIR /home/${user}
 
+FROM build as default
```

```suggestion
FROM build AS default
```
I'm being nit-picky.